src/GuppyDataExplorer/GuppyDataExplorer.jsx

```diff
@@ -31,6 +32,7 @@ class GuppyDataExplorer extends React.Component {
           const isValidJSON = IsValidJSONString(decodedFilter);
           if (isValidJSON) {
             initialFilter = JSON.parse(decodedFilter).filter;
+            hasInitialFilter = true;
```

Comment:
if we land on this page without url filters, is `isValidJSON` false? iirc an empty string is valid json 🤔

Reply:
if its empty it wont pass the check for stateFromURL